packages/graph/hash_graph/lib/graph/src/store/postgres/query/expression/join_clause.rs

```diff
@@ -19,12 +19,10 @@ impl Transpile for JoinExpression<'_> {
     fn transpile(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         // TODO: https://app.asana.com/0/1202805690238892/1203324626226299/f
         fmt.write_str("INNER JOIN ")?;
-
-        if self.join.alias.is_some() {
-            self.join.column.table().transpile(fmt)?;
-            fmt.write_str(" AS ")?;
-        }
-        self.join.table().transpile(fmt)?;
+        let table = self.join.table();
+        table.table.transpile(fmt)?;
+        fmt.write_str(" AS ")?;
+        table.transpile(fmt)?;
```

Comment:
Cool, On the former PR i was wondering whether or not it would make sense to always have the alias. I guess this answers that, thanks!